pennylane/interfaces/jax_jit_tuple.py

```diff
@@ -189,6 +191,7 @@ def wrapper(p):
             """Compute the forward pass."""
             new_tapes = set_parameters_on_copy_and_unwrap(tapes, p)
             res, _ = execute_fn(new_tapes, **gradient_kwargs)
+            res = list(res)
```

Comment:
What breaks if this is a tuple instead?

Reply:
`Expected tuple, got [ShapeDtypeStruct(shape=(), dtype=float64)]`

Reply:
This rather bothers me since I'm wanting to move more things to being tuples rather than lists, and I can't figure out why it would actually make a difference.

Reply:
I considered this, and tried instead to make everything a tuple, but it became very intrusive and made the work required a little too messy. Basically I found the code that returned a `list[ShapeDtypeStruct]` and made it return a tuple (also changed `_compute_jvps` in jax.py to return a tuple). Now, some things said `ValueError: Expected list, got (ShapeDtypeStruct(shape=(), dtype=float64),)`. I'm trying to drill all the way down and make everything a tuple, but it's quite hard since the error only occurs in jitted tests

Reply:
How about we convert all results to tuples instead?

If we:

1) Make `_tapes_shape_dtype_tuple` be a tuple
2) Convert all results objects to tuples on Line 194
3) Cast `jvps` to a tuple  on Line 281
4) cast results to tuple on Line 373
5) cast `jvps` to a tuple on line 445

Works for me, and I'd rather move to using tuples everywhere then using lists.

Reply:
ok it was a bit of a mess to track down where we need casting (because it spans into `jax` itself) but I did it! let's see if tests fail